$OpenBSD: patch-configure,v 1.3 2014/04/24 16:29:03 benoit Exp $

Use GNU readline as our base readline lacks add_history_time().

--- configure.orig	Fri Jan 24 08:59:53 2014
+++ configure	Sun Apr 13 08:55:09 2014
@@ -48459,7 +48459,7 @@ if ${lftp_cv_precompiled_readline+:} false; then :
 else
 
    old_LIBS="$LIBS"
-   LIBS="-lreadline $READLINE_SUPPLIB $LIBS"
+   LIBS="-lereadline $READLINE_SUPPLIB $LIBS"
    cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 extern int (*rl_getc_function)();
@@ -48499,7 +48499,7 @@ if test $lftp_cv_precompiled_readline = yes; then
    READLINE_DIR=''
    { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
-   READLINE='-lreadline'
+   READLINE='-lereadline'
    READLINE_DEPEND=''
    COMPILE_READLINE=no
 else
